src/index.js

```diff
@@ -32,28 +32,34 @@ class TelldusTDToolPlatform {
     this.log = log
     this.config = config
     this.homebridge = homebridge
+    this.senzors = config["sensors"] || [];
```

Comment:
Why not `sensors`? I'm also not sure why this would need to be a instance variable, when `this.devices` is not.